ydb/core/scheme/scheme_tablecell.h

```diff
@@ -499,6 +499,125 @@ class TSerializedCellVec {
     TVector<TCell> Cells;
 };
 
+class TOwnedCellVecBatch {
+public:
+    TOwnedCellVecBatch();
+
+    TOwnedCellVecBatch(const TOwnedCellVecBatch& rhs) = delete;
+
+    TOwnedCellVecBatch & operator=(const TOwnedCellVecBatch& rhs) = delete;
+
+    TOwnedCellVecBatch(const TOwnedCellVecBatch&& rhs) = default;
+
+    TOwnedCellVecBatch & operator=(TOwnedCellVecBatch&& rhs) = default;
+
+    size_t Append(TConstArrayRef<TCell> cells);
+
+    size_t Size() const {
+        return Cells.size();
+    }
+
+    bool Empty() const {
+        return Cells.empty();
+    }
+
+    bool empty() const {
+        return Cells.empty();
+    }
+
+    using iterator = TVector<TConstArrayRef<TCell>>::iterator;
+    using const_iterator = TVector<TConstArrayRef<TCell>>::const_iterator;
+
+    iterator begin() {
+        return Cells.begin();
+    }
+
+    iterator end() {
+        return Cells.end();
+    }
+
+    const_iterator cbegin() {
+        return Cells.cbegin();
+    }
+
+    const_iterator cend() {
+        return Cells.cend();
+    }
+
+    TConstArrayRef<TCell> operator[](size_t index) const {
+        return Cells[index];
+    }
+
+    TConstArrayRef<TCell> front() const {
+        return Cells.front();
+    }
+
+    TConstArrayRef<TCell> back() const {
+        return Cells.back();
+    }
+
+private:
+    static constexpr size_t InitialBufferSizeDegree = 16;
+
+    class Buffer {
```

Comment:
Classes should start with `T`, i.e. `TBuffer`.

Reply:
Updated.